src/plugins/kibana/server/lib/ingest/processors/append/converter.js

```diff
@@ -9,11 +11,15 @@ export default {
     };
   },
   esToKibana: function (processorEsDocument) {
+    if (!_.has(processorEsDocument, 'append')) {
+      throw new Error('Source object missing [append] property');
```

Comment:
Do you think the error would be more understandable in stack traces if we changed `Source object` to `Elasticsearch processor document` or maybe just `Processor document`?